.github/workflows/pepr-excellent-examples.yml

```diff
@@ -142,7 +142,7 @@ jobs:
     if: needs.examples-matrix.outputs.matrix != ''
     strategy:
       fail-fast: false
-      max-parallel: 5
+      max-parallel: 32 # Roughly matches the number of E2E tests and below GitHub concurrency limit
```

Comment:
Should we set it to a little over our current number of e2e tests since we are constantly adding new e2e tests?

Reply:
I chose a power of 2 that was greater than 24. We could conceivably run into parallelization issues if we have _too_ many tests triggered at once (but haven't... yet). GitHub limits us to [500 concurrent jobs](https://docs.github.com/en/enterprise-cloud@latest/actions/administering-github-actions/usage-limits-billing-and-administration#usage-limits) on smaller runners. I didn't want to set it to `500` since I'm pretty sure we'd run into runner resource issues before we hit that point.

32 is a level that we probably won't hit for awhile (how often do we add new E2E tests?) and the comment hopefully explains the "magic number."

Reply:
We seem to add new e2e tests on a regular basis. I just wanted to make sure that we don't have to keep coming in and changing the number all the time.

Reply:
It's unlikely that we will. In a scenario where we have 33 E2E tests, the 33rd one will get scheduled as soon as any other test finishes.

In a world where we have 64 E2E tests, we'll probably feel any CI performance pain points before we get there.